server/src/controllers/setlist.controllers.ts

```diff
@@ -0,0 +1,105 @@
+import { Request, RequestHandler, Response } from 'express';
+import { Setlist, ISetlist } from '../models/setlist.model';
+
+const sendResponse = (
+  res: Response,
+  statusCode: number,
+  payload: ISetlist[] | ISetlist | string
```

Comment:
might be a problem cos our setlist interface doesn't have the ID field, and the documents returned by the MongoDB operations will have it...